CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff

```diff
@@ -720,14 +754,17 @@
       </trans-unit>
       <trans-unit id="Image" xml:space="preserve">
         <source>Image</source>
+        <target>Zdięcie</target>
```

Comment:
"Zdjęcie", ale nie znam kontekstu, może być to też obraz albo plakat

Reply:
@cliss What kind of image is meant here?

Reply:
Took me a minute to find this one! 🤪 

This is used when you tap-and-hold on a _zoomed_ image — a actor's headshot or a movie/show poster — and attempt to share it. It would only reasonably be shown when generating a thumbnail for an image didn't work. Which should never happen

…in theory. 🫣